modules/cnf-installing-sriov-fec-operator.adoc

```diff
@@ -0,0 +1,170 @@
+
+The FEC devices are provided by the Intel® FPGA PAC N3000 FPGA and the Intel® vRAN Dedicated Accelerator ACC100 for the vRAN use case.
+
+[NOTE]
+====
+The Intel® FPGA PAC N3000 FPGA requires flashing with an 4G/LTE or 5G bitstream.
+====
+
+The OpenNESS SR-IOV Operator for Wireless FEC Accelerators provides functionality to create desired virtual functions (VFs) for the FEC device, binds them to appropriate drivers, and configures the VFs queues for desired functionality in 4G/LTE or 5G deployment.
+
+As a cluster administrator, you can install the OpenNESS SR-IOV Operator for Wireless FEC Accelerators by using the {product-title} CLI or the web console.
+
+[id="installing-sriov-fec-operator-using-the-cli_{context}"]
+== Installing the OpenNESS SR-IOV Operator for Wireless FEC Accelerators by using the CLI
+
+As a cluster administrator, you can install the OpenNESS SR-IOV Operator for Wireless FEC Accelerators by using the CLI.
+
+.Prerequisites
+
+* A cluster installed on bare-metal hardware.
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a namespace for the OpenNESS SR-IOV Operator for Wireless FEC Accelerators by completing the following actions:
+
+.. Define the `vran-acceleration-operators` namespace by creating a file named `sriov-namespace.yaml` as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: vran-acceleration-operators
+    labels:
+       openshift.io/cluster-monitoring: "true"
+----
+
+.. Create the namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-namespace.yaml
+----
+
+. Install the OpenNESS SR-IOV Operator for Wireless FEC Accelerators in the namespace you created in the previous step by creating the following objects:
+
+.. Create the following `OperatorGroup` CR and save the YAML in the `sriov-operatorgroup.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+    name: vran-operators
+    namespace: vran-acceleration-operators
+spec:
+    targetNamespaces:
+      - vran-acceleration-operators
+----
+
+.. Create the `OperatorGroup` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-operatorgroup.yaml
+----
+
+.. Run the following command to get the `channel` value required for the next step.
++
+[source,terminal]
+----
+$ oc get packagemanifest sriov-operator -n openshift-marketplace -o jsonpath='{.status.defaultChannel}'
+----
++
+.Example output
+[source,terminal]
+----
+stable
+----
+
+.. Create the following `Subscription` CR and save the YAML in the `sriov-sub.yaml` file:
```

Comment:
Revise to be parallel with the preceding instrucitons or change the first instruction to be parallel with this one and the preceding instruction.  My $0.02 is that omitting the backticks and abbreviations aids readability.

```
Define the subscription for the operator by creating a file named `sriov-sub.yaml` as shown in the following example:
```

Reply:
I am following the same structure as shown here https://docs.openshift.com/container-platform/4.8/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html which I have stepped through on my local machine and made sense